### PR TITLE
 Swedish: Add access keys

### PR DESCRIPTION
**Pull request by magol**
Add access keys to Swedish
Make two check boxes a bit wider so that Swedish text fit.

### CODE REVIEW

File: PowerEditor/installer/nativeLang/swedish.xml [hunk 1]
```diff
@@ -343,49 +343,49 @@
...
 				<Item id="1687" name="Vid förlorat fokus"/>
 				<Item id="1688" name="Alltid"/>
 				<Item id="1632" name="I mar&amp;kering"/>
 				<Item id="1633" name="Rensa alla markeringar"/>
-				<Item id="1635" name="Ersätt allt i alla öppna dokument"/>
-				<Item id="1636" name="Sök allt i alla öppna dokument"/>
-				<Item id="1654" name="Filter:"/>
-				<Item id="1655" name="Mapp:"/>
+				<Item id="1635" name="Ersätt allt i alla &amp;öppna dokument"/>
+				<Item id="1636" name="Sök allt i alla ö&amp;ppna dokument"/>
```
Comment (ordtrogen) [inline PowerEditor/installer/nativeLang/swedish.xml]: While you're at it, take a look at the Sök-dialog and consider whether "Sök allt" shouldn't be "Sök alla" :-)
I was just about to submit a PR and noticed you were already working in this file, so if you like, you can fix this.

File: PowerEditor/installer/nativeLang/swedish.xml [hunk 2]
```diff
@@ -343,49 +343,49 @@
...
 				<Item id="1609" name="Ersätt &amp;alla"/>
 				<Item id="1687" name="Vid förlorat fokus"/>
 				<Item id="1688" name="Alltid"/>
 				<Item id="1632" name="I mar&amp;kering"/>
 				<Item id="1633" name="Rensa alla markeringar"/>
-				<Item id="1635" name="Ersätt allt i alla öppna dokument"/>
-				<Item id="1636" name="Sök allt i alla öppna dokument"/>
-				<Item id="1654" name="Filter:"/>
-				<Item id="1655" name="Mapp:"/>
+				<Item id="1635" name="Ersätt allt i alla &amp;öppna dokument"/>
```
Comment (ordtrogen) [inline PowerEditor/installer/nativeLang/swedish.xml]: "Ersätt allt" --> "Ersätt alla"  (see other comment)

**Comment by ordtrogen** [comment]
Assuming that OP has check for collisions of access keys within menus, this PR looks completely fine and I approve merging it. Is there an "approve" button I should push?

+approve

**Comment by magol** [comment]
@ordtrogen  
Thanks for the input.
I agree that we should be consistent with "alla" and "allt". But I don't like constructions like "Sök alla i alla öppna dokument", I think "Sök allt i alla öppna dokument" sound better. 

But, if you don't agree with me, I can change it to "alla" :-)

**Comment by ordtrogen** [comment]
What is it you don't like about them?

**Comment by magol** [comment]
What i don't like is that "alla" is used twice in the same  sentence and refering to different things (all string matches and all files).

But i can live with it 😊

**Comment by ordtrogen** [comment]
This went too fast for me. But maybe I'll open a PR and we can discuss my suggestions there. Thanks for the access keys anyway

**Comment by magol** [comment]
@ordtrogen Yes, I was surprised to, but now we got the access keys. Please create a new PR and reference me, so can we continue the discussion :-)

**Pull request closed without merging**